### PR TITLE
Created PlayerController

### PR DESCRIPTION
**Pull request by angelika-c**
App can search player by nickname. Questions in comments.
I had a general problem with naming classes/packages. I don't know the correct schemas to do it.

### CODE REVIEW

File: src/main/java/com/github/fairit/be_doyou_destinypro/controller/PlayerController.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+public class PlayerController {
+
+    private PlayerService playerService;
+
+    @Autowired
+    public PlayerController(final PlayerService playerService) {
+        this.playerService = playerService;
+    }
+
+    @PostMapping("/find")
```
Comment (angelika-c) [inline src/main/java/com/github/fairit/be_doyou_destinypro/controller/PlayerController.java]: Sample JSON for post method:
`{
	"nickname":"katojido"
}`

File: src/main/java/com/github/fairit/be_doyou_destinypro/service/player_service/PlayerApiService.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Autowired
+    public PlayerApiService(final RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    public PlayerApi findPlayerApiByNickname (String nickname){
+
+        String addressURL = currentApiAddress.replace("{nickname}", nickname);
+        HttpHeaders headers = new HttpHeaders();
```
Comment (angelika-c) [inline src/main/java/com/github/fairit/be_doyou_destinypro/service/player_service/PlayerApiService.java]: I found this method to add headers on the internet but I don't like it. How to change this method to hide `api-key`? Maybe there is a better way to do it.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1 +1,9 @@
+api.bungie.address.player=https://www.bungie.net/Platform/Destiny2/SearchDestinyPlayer/-1/{nickname}/
 
+spring.datasource.url=jdbc:mysql://remotemysql.com:3306/T3qRnlKrf3
```
Comment (angelika-c) [inline src/main/resources/application.properties]: I was forced to add these properties. Draft version.

File: src/main/java/com/github/fairit/be_doyou_destinypro/controller/PlayerController.java [hunk 4]
```diff
@@ -1,25 +1,30 @@
...
-    public PlayerController(final PlayerService playerService) {
+    public PlayerController(final PlayerService playerService, final CharacterApiService characterApiService) {
         this.playerService = playerService;
+        this.characterApiService = characterApiService;
     }
 
-    @PostMapping("/find")
-    public PlayerApi searchPlayerByNickname(@RequestBody Player player) {
-        return playerService.findByPlayerNickname(player);
+    @RequestMapping(value = "/findplayer/{nickname}", method = POST)
```
Comment (angelika-c) [inline src/main/java/com/github/fairit/be_doyou_destinypro/controller/PlayerController.java]: I don't sure it's good way to do it. `@PostMapping` `@RequestMapping` or something else?

Comment (pbielicki) [reply src/main/java/com/github/fairit/be_doyou_destinypro/controller/PlayerController.java]: @RequestMapping is OK

File: src/main/resources/application.properties (hunk 3 above)
Comment (pbielicki) [reply src/main/resources/application.properties]: 👍  in "prod" use environment variable

**Pull request merged**